elasticsearch_dsl/search.py

```diff
@@ -616,9 +616,23 @@ def suggest(self, name, text, **kwargs):
 
             s = Search()
             s = s.suggest('suggestion-1', 'Elasticsearch', term={'field': 'body'})
+
         """
+        if text is None and regex is None:
+            raise ValueError('You have to pass "text" or "regex" argument.')
```

Comment:
I think we need additional validation rules here - if you supply both `text` and `regex` or you supply `regex` and `completion` is not in `kwargs` etc - any invalida combination essentially should raise a `ValueError`.

Reply:
OK, i'll fix it more strictly.